app.js

```diff
@@ -1 +1,24 @@
-const express = require('express')
\ No newline at end of file
+//Server file
+
```

Comment:
The correct naming convention for your server file should be named server.js, not app.js. This is also confusing as you have an App.js in the client folder.